Dockerfile

```
@@ -0,0 +1,19 @@
+FROM golang:1.17.3
```

Would it be better to omit the patch version?
```suggestion
FROM golang:1.17
```

---

OK.